generator/generator.go

```diff
@@ -245,6 +246,94 @@ func buildMessagesFromTypes(descr *dpb.FileDescriptorProto, renderer *Renderer)
 	return nil
 }
 
+// adjustMethodsAndTypes simplifies and prettifies the types and fields of the surface model in order to get a better
+// looking output file.
+// Related to: https://github.com/googleapis/gnostic-grpc/issues/11
+func adjustMethodsAndTypes(renderer *Renderer) {
```

Comment:
This is an interesting approach - it essentially adds another "pipeline stage" that prepares the surface model for the next generation stage by applying the assumptions specific to our gRPC model generation. No action needed, but as we add features I want to watch this to see if it's better to do this with a copy-and-modify or by direct construction instead of an in-place transformation.

Reply:
Interesting is not always optimal, so feel free make suggestions :)

Yes exactly. I believe something similar is necessary for gnostic-go-generator.